types/office-js-preview/index.d.ts

```diff
@@ -89332,6 +89332,8 @@ declare namespace Word {
          * @remarks
          * [Api set: WordApi 1.1]
          *
```

Comment:
there are several more insertFileFromBase64 APIs on object such as body, document, contentcontrol, range, inlinePicture, paragraph.... should we add for all of them?

Reply:
if it applies to them, certainly I can do that

Reply:
yes, it applies to all of them.